tui/tui.go

```diff
@@ -318,3 +326,94 @@ func (tuiCore *TUICore) ExitCommandMode(force bool) bool {
 
 	return false
 }
+
+var spinners = [...]string{"⠋", "⠙", "⠹", "⠸", "⠼", "⠴", "⠦", "⠧", "⠇", "⠏"}
```

Comment:
I think this is the best option but I have no attachment either way if you wanna change.